en/docs/genai/develop/agents/observability.md

```
@@ -1,171 +1,228 @@
+
+3. In `Config.toml`, enable tracing and select the provider:
+
+    ```toml
+    [ballerina.observe]
+    tracingEnabled = true
+    tracingProvider = "jaeger"
+    ```
+
+When you run the integration, you should see a startup log line confirming the provider is wired up:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)